Library/Homebrew/sorbet/rbi/gems/logger@1.6.4.rbi

```diff
@@ -1,940 +0,0 @@
-# typed: false
```

Comment:
Note that this was being marked `false` anyway, presumably due to conflicts with the sorbet version. (Might be worth finding the diff and upstreaming any fix, if someone feels motivated…)